docs/server/ask-ellie.md

```
@@ -104,6 +106,113 @@ table describes the available tools:
+The `delete_memory` tool removes a memory by its
+numeric ID. A user can only delete memories that the
+user owns.
+
+### Example Interactions
+
+The following examples show how to use chat memory
+with Ellie.
+
+To store a preference, send a message such as:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)